.github/workflows/artifact.yml

```diff
@@ -18,7 +18,8 @@ jobs:
         uses: actions/setup-node@v4.0.2
         with:
           node-version: 20
-          registry-url: 'https://npm.pkg.github.com'
+          # TODO: figure out why this is failing in github actions
```

Comment:
We had this so we could publish to our Github registry: https://github.com/orgs/dbos-inc/packages

However, we might not want to publish to this registry anymore.

Reply:
Here's my hypothesis: This PR seems to have a huge change to the `package-lock.json` file. If you look at the diff here, the following lines are deleted. Now npm tries to find the package from the configured "registry-url" instead of the actual url from npmjs.org.

```
      "resolved": "https://registry.npmjs.org/@dbos-inc/dbos-sdk/-/dbos-sdk-1.13.10.tgz",
      "integrity": "sha512-SwSP+HEAS4siaErRejJ+ZyLqwSUR480h7qnkvxfVWCOQX/NPW0ScpwoBq7ndYqDwYEJXiR+E0UbUbibBtv8Pkg==",
```
https://github.com/dbos-inc/dbos-transact/pull/486/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519L1708-L1709

Reply:
I'll try fixing that! Great eye!

Reply:
THis worked! I copied package-lock over from main and reran `npm install`. Created a much simpler diff and left the resolved/integrity fields @qianl15 called out